devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1822,18 +1822,6 @@ bool WholeBodyDynamicsDevice::attachAllFTs(const PolyDriverList& p)
         return false;
     }
 
-    if (nrMASFTSensors != remappedMASInterfaces.ftMultiSensors->getNrOfSixAxisForceTorqueSensors())
-    {
-        yError() << "WholeBodyDynamicsDevice::attachAll Invalid number of MAS FT sensors after remapper";
-        return false;
-    }
-
-    if (nrMASFTSensors != ftMultipleAnalogSensorNames.size())
-    {
-	yError() << "WholeBodyDynamicsDevice::attachAll Invalid number of MAS FT sensor names";
-        return false;
-    }
```

Comment:
Should we check that at least ``nrMASFTSensors`` is lower?

Reply:
Until we solve https://github.com/robotology/whole-body-estimators/issues/32 this check is basically https://github.com/robotology/whole-body-estimators/pull/193/files#diff-938e8a6e8fc4690975a0955e2f3a91a8a23c313180dda2dea0fc6cbea41046d2R1806-R1813 .